File: PersonnelManagement/PersonnelManagement/Front/classes/Requests_classes/rank_request.ts
﻿import StructureTree from '../../classes/structuretree';

export default class Rank_Request {
    constructor() {
        this.current_structure = [];
        this.last_rank_date = new Date();
        this.corelate_rank = false;
    }
    current_structure: StructureTree[];
    last_rank_date: Date;
    corelate_rank: boolean;
}

File: PersonnelManagement/PersonnelManagement/Models/Rank_request.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PersonnelManagement.Models
{
    public partial class Rank_request
    {
        public IEnumerable<StructureTree> Current_structure { get; set; }
        public DateTime Last_rank_date { get; set; }
        public bool Corelate_rank { get; set; }
    }
}
